src/test/scala/org/apache/spark/deploy/k8s/integrationtest/backend/GCE/GCETestBackend.scala

```diff
@@ -20,20 +20,22 @@ import io.fabric8.kubernetes.client.{ConfigBuilder, DefaultKubernetesClient}
 
 import org.apache.spark.deploy.k8s.integrationtest.Utils
 import org.apache.spark.deploy.k8s.integrationtest.backend.IntegrationTestBackend
-import org.apache.spark.deploy.k8s.integrationtest.config._
 
-private[spark] class GCETestBackend(val master: String) extends IntegrationTestBackend {
+private[spark] class GCETestBackend() extends IntegrationTestBackend {
```

Comment:
Make this an `object` similar to `MinikubeTestBackend`.

Reply:
Also ideally we can remove this abstraction, i.e. the integration tests are just given a master URL and are agnostic to the backend type and don't run the Minikube binary. But for another day.

Reply:
Done.